src/1-Profile/Edit-Profile.tsx

```diff
@@ -158,11 +173,25 @@ const EditProfile = ({navigation}:StackNavigationProps):JSX.Element => {
             <Text style={styles.header}>Edit Profile</Text>
             <ScrollView>
               {renderInputFields()}
+              <Outline_Button buttonStyle={styles.imageUploadButton}
+                text={"Change Profile Picture"}
+                textStyle={styles.imageUploadButtonText}
+                
+                onPress={() => setProfileImageUploadModalVisible(true)}
+            />
             </ScrollView>
             <Raised_Button buttonStyle={styles.sign_in_button}
                 text='Save Changes'
                 onPress={handleSubmit(onEditFields)}
             />
+            <Modal
+              visible={profileImageUploadModalVisible}
+              onRequestClose={profileImageUploadCallback}
+            >
+              <ProfileImageUpload 
```

Comment:
Let's also have the ability to delete/clear profile image.
`DELETE /api/user/{{user-id}}/image`